.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
+        name: Install Python
+        with:
+          python-version: "3.x"
+          allow-prereleases: false
+
+      - name: Build distribution via nox
+        run: pipx run nox --error-on-missing-interpreters -s release_build
+
+      - name: Upload distribution
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # 4.6.2
+        with:
+          name: packages
+          path: dist/
+          if-no-files-found: error
+          compression-level: 0
```

Comment:
Does this improve performance?

Reply:
I'm generally morally opposed to compressing already-compressed files, regardless of performance. In this case however, I would suspect this to improve performance; [the documentation](https://github.com/actions/upload-artifact?tab=readme-ov-file#altering-compressions-level-speed-v-size) says so.

Reply:
Oh, TIL